creating_packages/toolchains/cmake.rst

```diff
@@ -0,0 +1,168 @@
+CMakeToolchain
+==============
+
+The ``CMakeToolchain`` can be used in the ``toolchain()`` method:
+
+
+.. code:: python
+
+    from conans import ConanFile, CMake, CMakeToolchain
+
+    class App(ConanFile):
+        settings = "os", "arch", "compiler", "build_type"
+        requires = "hello/0.1"
+        generators = "cmake_find_package_multi"
+        options = {"shared": [True, False], "fPIC": [True, False]}
+        default_options = {"shared": False, "fPIC": True}
+
+        def toolchain(self):
+            tc = CMakeToolchain(self)
+            return tc
+
+
+The ``CMakeToolchain`` will generate 2 files:
```

Comment:
```suggestion
The ``CMakeToolchain()`` will generate 2 files:
```

Reply:
That is the name of the toolchain itself, creating the object, will not generate 2 files.